pkg/storage/internalstorage/config.go

```diff
@@ -48,9 +58,30 @@ type Config struct {
 
 	Params map[string]string `yaml:"params"`
 
+	AutoMigration  *bool            `yaml:"autoMigration"` // If set to false, no tables will be created
```

Comment:
SkipAutoMigration would probably be more appropriate and would not require the `*bool`